package.json

```diff
@@ -72,11 +72,12 @@
     "webpack-cli": "^3.3.11"
   },
   "dependencies": {
-    "@deck.gl/core": "8.1.7",
-    "@deck.gl/geo-layers": "8.1.7",
-    "@deck.gl/google-maps": "8.1.7",
-    "@deck.gl/layers": "8.1.7",
-    "@deck.gl/mesh-layers": "8.1.7",
+    "@deck.gl/core": "8.1.9",
```

Comment:
Nice, this one includes @juandjara contribution!!
Very soon we'll upgrade to alpha.3